week-03-ajax-json-express/day-01-intro-ajax/dawn-forms/README.md

```diff
@@ -0,0 +1,154 @@
+| text area | `<textarea></textarea>` | ![<textarea></textarea>][area] | a more customizable plain text area |
+| checkbox | `<input type="checkbox">` | ![<input type="checkbox">][check] | can be toggled on or off |
+| radio button | `<input type="radio">` | ![<input type="radio" name="group"> <input type="radio" name="group">][radio] | can be grouped with other inputs |
+| drop-down lists | `<select><option>` | ![<select><option>Option 1</option><option>Option 2</option></select>][select] | [check here for more info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) |
+| file picker | `<input type="file">` | ![<input type="file">][file] | pops up an “open file” dialog |
+| hidden field | `<input type="hidden">` |  | nothing there!
+| submit button | `<input type="submit">` | ![<input type="submit">][submit] | activates the form's submission <br/>(a `POST` request or <br/>Javascript action) |
+
+<!-- Images -->
+[text]:   https://raw.github.com/h4w5/html_form_cheatsheet_images/master/input-text.png
+[area]:   https://raw.github.com/h4w5/html_form_cheatsheet_images/master/textarea.png
+[check]:  https://raw.github.com/h4w5/html_form_cheatsheet_images/master/input-checkbox.png
+[radio]:  https://raw.github.com/h4w5/html_form_cheatsheet_images/master/input-radio.png
+[select]: https://raw.github.com/h4w5/html_form_cheatsheet_images/master/select-option.png
+[file]:   https://raw.github.com/h4w5/html_form_cheatsheet_images/master/input-file.png
+[submit]: https://raw.github.com/h4w5/html_form_cheatsheet_images/master/input-submit.png
+
+### Important Attributes
+
+All input types (including `<textarea>`s):
+
+- **`type`**: the type of data that is being input (affects the "widget" that is used to display this
+  element by the browser).
+- **`name`**: the key used to describe this data in the HTTP request.
+- **`id`**: the unique identifier that other HTML elements, JavaScript and CSS use to access this
+  element in the browser.
+- **`value`**: the default data that is assigned to the element.
+- **`placeholder`**: not a default value, but a useful HTML5 addition of a data "prompt" for an input.
+- **`autofocus`**: defaults the cursor to a specific input when the page originally loads. You can only have one autofocus on your page.
+- **`disabled`**: a Boolean attribute indicating that the "widget" is not available for interaction.
+
+Radio buttons or checkboxes:
+
+- **`checked`**: a Boolean that indicates whether the control is selected by default (is false unless).
+- **`name`**: the group to which this element is connected. For radio buttons, only one element per
+  group (or name) can be checked.
+- **`value`**: the data or value that is returned for a specific group (a multi-element control), if
+  this element is checked.
+
+## Common Validations
+
+Validations avoids having users submit bad data to our application. Knowing how to use them will save time and make your app a lot more usable.
```

Comment:
Validations avoid  (no s)

Or maybe "Validations help to prevent users from submitting bad data to the server.